ci/build_python_pypi.sh

```diff
@@ -8,11 +8,36 @@ python -m pip install build --user
 export GIT_DESCRIBE_TAG=$(git describe --abbrev=0 --tags)
 export GIT_DESCRIBE_NUMBER=$(git rev-list ${GIT_DESCRIBE_TAG}..HEAD --count)
 
+# Build date for PyPI pre-releases
+export RAPIDS_VERSION_NUMBER="23.08"
+export RAPIDS_FULL_VERSION_NUMBER="${RAPIDS_VERSION_NUMBER}.00"
+if rapids-is-release-build; then
+  export PACKAGE_VERSION_NUMBER="${RAPIDS_FULL_VERSION_NUMBER}"
+else
+  export BUILD_DATE=$(date +%y%m%d)
+  export PACKAGE_VERSION_NUMBER="${RAPIDS_FULL_VERSION_NUMBER}a${BUILD_DATE}"
+fi
+
 # Compute/export RAPIDS_DATE_STRING
 source rapids-env-update
 
+# Inplace sed replace; workaround for Linux and Mac
+function sed_runner() {
+    sed -i.bak ''"$1"'' $2 && rm -f ${2}.bak
+}
```

Comment:
Let's avoid proliferating the `sed_runner` function. It has problems, and we don't need it here. This script is only ever run in CI on Linux machines. Let's just use plain sed for now.

Reply:
Done in https://github.com/rapidsai/dask-cuda/pull/1204/commits/d4d887a9780664bacf077f7a5567c7a81592289a .